src/book/03-guides/04-general/01-learning-resources.mdx

```diff
@@ -0,0 +1,180 @@
+
+**Videos:**
+
+- [LinkedIn Learning: C++ Basics](https://www.linkedin.com/learning/c-plus-plus-essential-training) - For student who have not completed [SENG1120](https://www.newcastle.edu.au/course/SENG1120), or need a refresher.
+- [LinkedIn Learning: C++ Advanced Topics](https://www.linkedin.com/learning/c-plus-plus-advanced-topics?u=75842122)
+
+**Tutorials:**
+
+- NUbots runs [a workshop on modern C++](https://github.com/NUbots/NULearning/blob/master/cpp/moderncpp.md).
+
+### Lab Induction
+
+The lab induction must be completed before getting swipe card access to the lab.
+
+**Reading:**
+
+- [NUbook: Lab Induction](/team/induction)
+
+**Tutorials:**
+
+- NUbots lab inductions are run for new team members after they join.
+
+### Using Webots
+
+The Webots simulator is used in some RoboCup competitions and is the simulator we use to aid in development of the robots.
+
+**Reading:**
+
+- [Setting up Webots](/guides/tools/webots-setup)
+
+### Linear Algebra
+
+Linear Algebra is important for many robotics systems. In the main robot codebase, the [Eigen C++ linear algebra library](http://eigen.tuxfamily.org/dox/) is used.
+
+**Reading:**
+
+- [NUbook: Mathematics](/system/foundations/mathematics#linear-algebra)
+
+### Log and Play Back Data
+
+**Reading:**
+
+- [Data Recording and Playback Guide](/guides/main/data-recording-playback)
+- [Logging](/system/subsystems/logging)
+
+### Python
+
+Python is used in the build system in the main robot codebase.
+
+**Videos:**
+
+- [LinkedIn Learning: Python Essential Training](https://www.linkedin.com/learning/python-essential-training-2/)
+
+### RoboCup Competition
+
+**Reading:**
+
+- [RoboCup Humanoid League: Rules of the Game](https://humanoid.robocup.org/materials/rules/)
+
+**Videos:**
+
+- [YouTube: Referee Training Video](https://www.youtube.com/watch?v=HHOtAklrcJU)
+
+## DevOps and Quality Assurance
+
+### The Build System
+
+**Reading:**
+
+- [NUbook: Build System](https://nubook.nubots.net/system/foundations/build-system)
+
+### Docker
+
+**Reading:**
+
+- [Book: Docker Deep Dive](https://www.amazon.com.au/Docker-Deep-Dive-Nigel-Poulton-ebook/dp/B01LXWQUFF)
+
+## DevTools
+
+### Web Development
+
+Web Development is used in the creation of [NUbook](/system/tools/nubook), which uses JavaScript and Gatsby, and [NUsight](/system/tools/nusight), which uses TypeScript and React. By working on these systems and going through these resources, you will gain valuable real world experience that is often sought out in industry.
+
+**Videos:**
+
+- [egghead.io: MobX](https://egghead.io/courses/manage-complex-state-in-react-apps-with-mobx)
+
+**Tutorials:**
+
+- [The Modern JavaScript Tutorial](https://javascript.info/)
+- [Official Node.js Tutorial](https://nodejs.dev/learn)
+- [egghead.io: A Beginner's Guide to React](https://egghead.io/courses/the-beginner-s-guide-to-react)
+- [Official Webpack Tutorial](https://webpack.js.org/guides/getting-started/)
+
+**Mixed:**
```

Comment:
Not sure where to put these, so I made this section. Maybe just put them all under 'Tutorials'?

Reply:
Maybe "Resources" ?

Reply:
Hm maybe, but everything would fit under that. I did consider 'Miscellaneous' or 'Other' or 'Extra'.